packages/vue-client/src/components/GameCreation/GameCreationForm.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
```

Comment:
Dear Git, why don't you recognize this file isn't new, but was just moved and modified?
See [How to make git mark a deleted and a new file as a file move?](https://stackoverflow.com/a/433142) if you want to avoid this in the future.

Reply:
Haha the diff is pretty significant, may have been unavoidable 😅